src/features/rewards/tableAdsHistory/style.ts

```diff
@@ -0,0 +1,171 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License. v. 2.0. If a copy of the MPL was not distributed with this file.
+* You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import styled from 'styled-components'
+
+export const StyledTHOther = styled<{}, 'th'>('th')`
+  text-transform: uppercase;
+  text-align: left;
+  font-family: ${p => p.theme.fontFamily.body};
+  font-size: 12px;
+  font-weight: 500;
+  border-bottom: 2px solid #dedfe4;
+  border-top: 2px solid #dedfe4;
+  color: #4B4C5C;
+  padding: 12px 0;
+  width: 70%;
+`
+
+export const StyledOuterTD = styled<{}, 'td'>('td')`
+  width: 100%;
+`
+
+export const StyledTR = styled<{}, 'tr'>('tr')`
+  text-align: left;
+`
+
+export const StyledTHLast = styled(StyledTHOther)`
+  text-transform: uppercase;
+  text-align: left;
+  font-family: ${p => p.theme.fontFamily.body};
+  font-size: 12px;
+  font-weight: 500;
+  border-bottom: 2px solid #dedfe4;
+  border-top: 2px solid #dedfe4;
+  color: #4B4C5C;
+  padding: 12px 0;
+  width: 15%;
+`
+
+export const StyledTHFirst = styled<{}, 'th'>('th')`
```

Comment:
what is the difference between `StyledTHFirst` and `StyledTHLast`. Also it looks like `StyledTHOther` and `StyledTHFirst` only have `width` different, so not sure why we duplicate everything else.

Reply:
Enough has changed to render these unnecessary. Removed, and replaced with single instance.